### ISSUE TITLE
Locale-specific failures

### ISSUE THREAD

**Issue by eserte** [cpanel/Test2-Harness-Renderer-JUnit#10]
t/test.t may fail on systems with non-English locale. Probably only older perls (< 5.20 or so) are affected:
```
    # Failed test 'No such file or directory.'
    # at t/test.t line 101.
    # +-------------------------------+----+-------------------------------+
    # | GOT                           | OP | CHECK                         |
    # +-------------------------------+----+-------------------------------+
    # | ( LAUNCH )  job  1    t/test- | =~ | (?^:No\ such\ file\ or\ direc |
    # | ok/pass-1.tx\n                |    | tory)                         |
    # | (  NOTE  )  job  1    Seeded  |    |                               |
    # | srand with seed '20201016' fr |    |                               |
    # | om local date.\n              |    |                               |
    # | [  PASS  ]  job  1  + pass\n  |    |                               |
    # | [  PLAN  ]  job  1    Expecte |    |                               |
    # | d assertions: 1\n             |    |                               |
    # | ( MEMORY )  job  1    rss:  N |    |                               |
    # | A\n                           |    |                               |
    # | ( MEMORY )  job  1    size: N |    |                               |
    # | A\n                           |    |                               |
    # | ( MEMORY )  job  1    peak: N |    |                               |
    # | A\n                           |    |                               |
    # | ( LAUNCH )  job  2    t/test- |    |                               |
    # | ok/pass-2.tx\n                |    |                               |
    # | ( PASSED )  job  1    t/test- |    |                               |
    # | ok/pass-1.tx\n                |    |                               |
    # | (  TIME  )  job  1    Startup |    |                               |
    # | : 0.38235s | Events: 0.00094s |    |                               |
    # |  | Cleanup: 0.00411s | Total: |    |                               |
    # |  0.38740s\n                   |    |                               |
    # | (  NOTE  )  job  2    Seeded  |    |                               |
    # | srand with seed '20201016' fr |    |                               |
    # | om local date.\n              |    |                               |
    # | [  PASS  ]  job  2  + pass\n  |    |                               |
    # | [  PLAN  ]  job  2    Expecte |    |                               |
    # | d assertions: 1\n             |    |                               |
    # | ( MEMORY )  job  2    rss:  N |    |                               |
    # | A\n                           |    |                               |
    # | ( MEMORY )  job  2    size: N |    |                               |
    # | A\n                           |    |                               |
    # | ( MEMORY )  job  2    peak: N |    |                               |
    # | A\n                           |    |                               |
    # | ( PASSED )  job  2    t/test- |    |                               |
    # | ok/pass-2.tx\n                |    |                               |
    # | (  TIME  )  job  2    Startup |    |                               |
    # | : 0.26109s | Events: 0.00125s |    |                               |
    # |  | Cleanup: 0.00305s | Total: |    |                               |
    # |  0.26539s\n                   |    |                               |
    # | Can't open '/var/tmp/cpansmok |    |                               |
    # | er-1023/2020101622/f4DcBrt_N0 |    |                               |
    # | /x/y/z/ok.xml' (Datei oder Ve |    |                               |
    # | rzeichnis nicht gefunden) at  |    |                               |
    # | /usr/perl5.14.4/lib/site_perl |    |                               |
    # | /5.14.4/Test2/Harness/Rendere |    |                               |
    # | r/JUnit.pm line 241, <$__ANON |    |                               |
    # | IO__> line 21.\n              |    |                               |
    # | \e[0m\n                       |    |                               |
    # +-------------------------------+----+-------------------------------+
    # Command = /usr/perl5.14.4/bin/perl /usr/perl5.14.4/bin/yath -D/usr/perl5.14.4/lib/site_perl/5.14.4 test -I/usr/perl5.14.4/lib/site_perl/5.14.4 t/test-ok --ext=tx --renderer=Formatter --renderer=JUnit -v
    # Exit = 512
    # ==== Output ====
    # ( LAUNCH )  job  1    t/test-ok/pass-1.tx
    # (  NOTE  )  job  1    Seeded srand with seed '20201016' from local date.
    # [  PASS  ]  job  1  + pass
    # [  PLAN  ]  job  1    Expected assertions: 1
    # ( MEMORY )  job  1    rss:  NA
    # ( MEMORY )  job  1    size: NA
    # ( MEMORY )  job  1    peak: NA
    # ( LAUNCH )  job  2    t/test-ok/pass-2.tx
    # ( PASSED )  job  1    t/test-ok/pass-1.tx
    # (  TIME  )  job  1    Startup: 0.38235s | Events: 0.00094s | Cleanup: 0.00411s | Total: 0.38740s
    # (  NOTE  )  job  2    Seeded srand with seed '20201016' from local date.
    # [  PASS  ]  job  2  + pass
    # [  PLAN  ]  job  2    Expected assertions: 1
    # ( MEMORY )  job  2    rss:  NA
    # ( MEMORY )  job  2    size: NA
    # ( MEMORY )  job  2    peak: NA
    # ( PASSED )  job  2    t/test-ok/pass-2.tx
    # (  TIME  )  job  2    Startup: 0.26109s | Events: 0.00125s | Cleanup: 0.00305s | Total: 0.26539s
    # Can't open '/var/tmp/cpansmoker-1023/2020101622/f4DcBrt_N0/x/y/z/ok.xml' (Datei oder Verzeichnis nicht gefunden) at /usr/perl5.14.4/lib/site_perl/5.14.4/Test2/Harness/Renderer/JUnit.pm line 241, <$__ANONIO__> line 21.
    # [0m
    # 
    # ========
```

**Comment by toddr** [comment]
Possibly failing with https://github.com/cpanel/Test2-Harness-Renderer-JUnit/runs/1275638589

**Comment by atoomic** [comment]
at first glance the two issues are not linked each other

**Comment by atoomic** [comment]
I cannot reproduce this issue using for example `LC_ALL=fr_FR prove -vl t/test.t` and prove 5.14
I wonder if this is fixed by #12

**Comment by toddr** [comment]
You're probably right this is an issue but not a priority for me until I see evidence someone is using yath on 5.14 :)

**Comment by eserte** [comment]
I still see the failures with 1.000003. And it's not <= 5.14, but <= 5.20. A sample report: http://www.cpantesters.org/cpan/report/641dee58-1251-11eb-a408-ae391f24ea8f

**Comment by toddr** [comment]
Yep. Waiting on cpantesters to come back to life then I'll take a closer look. Thanks for your vigilance @eserte !!
